src/styles/utils/_normalaize.scss

```diff
@@ -0,0 +1,4 @@
+body {
```

Comment:
The current rules in this file only reset padding and margin for the body element, but a complete CSS reset or normalize should include more rules to ensure consistent styling across browsers. Consider using a library like 'normalize.css' or add more rules to cover other elements as well.